packages/app/pages/organization/course/[cid]/edit.tsx

```diff
@@ -273,16 +270,10 @@ export default function Edit(): ReactElement {
                   <Col xs={{ span: 24 }} sm={{ span: 12 }}>
                     <Form.Item
                       label="Semester"
-                      name="semesterId"
+                      name="semesterName"
                       tooltip="Semester of the course"
                     >
-                      <Select>
-                        {semesters.map((semester) => (
-                          <Select.Option value={semester.id} key={semester.id}>
-                            {semester.season + semester.year}
-                          </Select.Option>
-                        ))}
-                      </Select>
```

Comment:
honestly I'm not sure if it matters, but is there a way to list all other semesters as well? That way, we don't have like multiple of the same semester (e.g. "Winter,2024" and "Winter,24", and maybe "winter,2024")

Reply:
Ramon specifically asked to allow professors to input any semester name or variation.

Reply:
oh i see, aight

Reply:
Yeah, that's my understanding. I am also okay with duplicate semesters as it would require a more complex logic to check for duplicates